src/pages/organizations/[organizationid].js

```diff
@@ -367,14 +367,14 @@ export default function Organization(props) {
                 ))}
                 {/* Placeholder, remove after API fixed */}
                 {organization?.species?.species?.map((s) => (
-                  <>
+                  <React.Fragment key={s.name}>
```

Comment:
I guess this is the same as: `<></>` ? right?

Reply:
Yes but this way you can pass the key